File: MiPrimeraAplicacion/app/src/main/java/com/micky/miprimeraaplicacion/GraficoActivity.java
package com.micky.miprimeraaplicacion;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.micky.miprimeraaplicacion.backend.Grafico;

public class GraficoActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        //setContentView(R.layout.activity_grafico);

        setContentView(new Grafico(GraficoActivity.this));
    }
}

File: PrimerKotlin/app/src/main/java/com/micky/primerkotlin/GraficoActivity.kt
package com.micky.primerkotlin

import android.annotation.SuppressLint
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import com.micky.primerkotlin.backend.Grafico

class GraficoActivity : AppCompatActivity() {
    @SuppressLint("MissingInflatedId")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(Grafico(this))

        /*var buttonSalirGrafico = Button(this)
        buttonSalirGrafico = findViewById(R.id.buttonSalirGrafico)
        buttonSalirGrafico.setOnClickListener {
            finish()
        }*/
    }

}